.github/workflows/CI.yaml

```diff
@@ -137,7 +152,9 @@ jobs:
           tar -xf /tmp/build_target.tar
 
       - name: "Build: Gear"
-        run: ./scripts/gear.sh build gear --release --locked
+        run: ./scripts/gear.sh build gear --locked --exclude "gear-test" --release
```

Comment:
Why `gear-test` has been excluded?

Reply:
Also get rid of `--release` flag across the job because we can get worse coverage report. And I expect build will be faster

Reply:
> Also get rid of `--release` flag across the job because we can get worse coverage report. And I expect build will be faster

As I can see the report didn't change when building in release mode and `gear-test` covers the same lines as `gear-node runtime-spec-test` https://codecov.io/github/gear-tech/gear/commit/44c3e40cf5136b200b930776e5d57f62a5082a49

Reply:
> As I can see the report didn't change when building in release mode

Percent is ok for the whole project but if you look closely you will see we lost ~10% of coverage for `core-backend` not speaking about other crates

> gear-test covers the same lines as gear-node runtime-spec-test

Should we remove `gear-test` at all then?

Reply:
`gear-test` is a lightweight tool to doublecheck execution logic in `wasmtime` backend. We can just ignore `wasmtime` coverage.

Reply:
> We can just ignore wasmtime coverage.

This won't help. We still have bad coverage. I just reprint what bot says:
> core-processor/src/common.rs	0.00% <0.00%> (-68.00%)
core-processor/src/ext.rs	74.54% <0.00%> (-14.10%)
core/src/costs.rs	73.68% <0.00%> (-7.90%)

This is absolutely unaccaptable

Reply:
moved `gear-test` to build job